pkg/urbit/include/vere/db/lmdb.h

```diff
@@ -1,5 +1,9 @@
-/* include/vere/db/lmdb-impl.h
+/* pkg/urbit/include/vere/db/lmdb-impl.h
```

Comment:
IDK how `-impl` got in here, but that should be removed.

Reply:
Removed the header path comments.